### DISCUSSION TITLE
is preferCacheValue the same as forceRefetch?

### DISCUSSION THREAD

**Discussion by davidjohnstoncoles** [reduxjs/redux-toolkit#4621] [Q&A] [answered]
The second argument of a lazyQueryTrigger is the boolean `preferCacheValue`. 

On the otherhand, if you were doing a endpoints.slice.initiate you could pass the property `forceRefetch` 

Are these functionally the same?

**Comment by EskiMojo14** [comment, marked as answer]
it's exactly the opposite:
https://github.com/reduxjs/redux-toolkit/blob/bc0bf0ffced8d1e9aa927faf81e37f42e681058c/packages/toolkit/src/query/react/buildHooks.ts#L961

preferCacheValue defaults to false, meaning it'll always force a refetch by default. Setting it true means that forceRefetch is false.

**Comment by davidjohnstoncoles** [comment]
^ Cheers, thanks a lot.
